pages/common/csv2tsv.md

```diff
@@ -0,0 +1,12 @@
+# csv2tsv
+
+> Convert CSV (comma-separated) text to TSV (tab-separated) format.
+> More information: <https://github.com/eBay/tsv-utils/blob/master/README.md#csv2tsv>.
+
+- Convert from CSV to TSV:
+
+`csv2tsv {{path/to/input_csv1 path/to/input_csv2 ...}} > {{path/to/output_tsv}}`
+
```

Comment:
I have a suggestion, this [page](https://github.com/eBay/tsv-utils/blob/master/docs/tool_reference/csv2tsv.md) specifies there are multiple field delimiters present like comma(default), etc. So maybe we can add a generalized command while having the `semicolon` example.
```suggestion

- Convert field delimiter separated CSV to TSV:

`csv2tsv -c'{{field_delimiter}}' {{path/to/input_csv}}`

```